packages/tpetra/core/src/Tpetra_BlockMultiVector_def.hpp

```diff
@@ -305,6 +305,7 @@ replaceLocalValuesImpl (const LO localRowIndex,
   auto X_dst = getLocalBlock (localRowIndex, colIndex);
   typename const_little_vec_type::HostMirror::const_type X_src (reinterpret_cast<const impl_scalar_type*> (vals),
                                                                 getBlockSize ());
+  // DEEP_COPY REVIEW - DEVICE-TO-DEVICE
```

Comment:
This is not device to device. It is host to device. In the above code, it uses HostMirror. 

This interface is not going to be efficient either. It provides an access to a single block in a block multi vector. If the block multi vector is defined on a gpu, then this causes frequent memory transfer for a single block. Since this interface defines on host, we should collect all modifications of small blocks. After all modifications are done, we should sync host view to device view. The syntax is probably....
```
block_multivector.beginModificationOnHost();
for (int i=0;i<many;++i) {
  block_multivector.replaceLocalValues(rowIdx, colIdx, valsFromHost);
}
block_multivector.endModificationOnHost();
```

Reply:
Fixed comment to be host-to-device. Thx.